flycheck.el

```diff
@@ -6777,18 +6777,14 @@ Each item is a string with a tag to be given to `go build'."
 (flycheck-define-checker go-build
   "A Go syntax and type checker using the `go build' command.
 
+Requires Go 1.6 or newer.
+
 See URL `https://golang.org/cmd/go'."
```

Comment:
Please fold both lines into a single paragraph.  Let's not use so much vertical space in docstrings, as the help buffers of syntax checkers are pretty stuffed with text already. :blush: